src/js/popup/popup.js

```diff
@@ -1736,6 +1739,32 @@
           },)
         }
 
+        // Show Holiday Promo 2023 News if the waffle flag is active and premium is available in user's country
+        if (isHolidayPromo2023Available && premiumAvailability) {
```

Comment:
When I run npm run config:prod, the news item does not show, I had to manually make it show by removing this if branch.

Edit: Whoops, this is because the waffle flag isn't active in prod